bigtable/google/cloud/bigtable/mutation.py

```diff
@@ -0,0 +1,125 @@
+            timestamp_micros=timestamp_micros,
+            value=value
+        )
+        mutation_message = data_v2_pb2.Mutation(set_cell=set_cell_mutation)
+        self.mutations.append(mutation_message)
+
+    def delete_cells(self, family_name, columns, time_range=None):
+        """Create the mutation request message for DeleteFromColumn and
+            add it to the list of mutations
+
+        Arguments:
+            family_name (str):
+                The name of the family into which new data should be written.
+                Must match ``[-_.a-zA-Z0-9]+``.
+            columns (list):
+                The columns within the column family that will have cells
+                deleted. If :attr:`ALL_COLUMNS` is used then the entire
+                column family will be deleted from the row.
+            time_range (TimestampRange):
+                (optional) The range of timestamps within which cells should be
+                deleted.
+        """
+        if columns is self.ALL_COLUMNS:
+            self.delete_from_family(family_name)
+        else:
+            for column_id in columns:
+                delete_from_column_mutation = (
+                    data_v2_pb2.Mutation.DeleteFromColumn(
+                        family_name=family_name,
+                        column_qualifier=_to_bytes(column_id),
+                        time_range=time_range))
+
+                mutation_message = data_v2_pb2.Mutation(
+                    delete_from_column=delete_from_column_mutation)
+                self.mutations.append(mutation_message)
+
+    def delete_from_family(self, family_name):
+        """Create the mutation request message for DeleteFromFamily and add
+        it to the list of mutations
+
+        Arguments:
+            family_name (str):
+                The name of the family into which new data should be written.
+                Must match ``[-_.a-zA-Z0-9]+``.
+        """
+        delete_from_family_mutation = data_v2_pb2.Mutation.DeleteFromFamily(
+            family_name=family_name
+        )
+        mutation_message = data_v2_pb2.Mutation(
+            delete_from_family=delete_from_family_mutation)
+        self.mutations.append(mutation_message)
+
+    def delete(self):
+        """Create the mutation request message for DeleteFromRow and add it
+        to the list of mutations"""
+        delete_from_row_mutation = data_v2_pb2.Mutation.DeleteFromRow()
+        mutation_message = data_v2_pb2.Mutation(
+            delete_from_row=delete_from_row_mutation)
+        self.mutations.append(mutation_message)
+
+    def create_entry(self):
+        """Create a MutateRowsRequest Entry from the list of mutations
+
+        Returns:
+            `Entry <google.bigtable.v2.MutateRowsRequest.Entry>`
+             An ``Entry`` for a MutateRowsRequest message.
+        """
+        entry = table_v2_pb2.MutateRowsRequest.Entry(row_key=self.row_key)
+        for mutation in self.mutations:
```

Comment:
We can skip this, if we create the `MutateRowsRequest.Entry` in the constructor, and set it to a member variable.  We could update t on each method, rather than doing extra work to copy from `self.mutations`.